modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/ChangePasswordPage.kt

```diff
@@ -0,0 +1,173 @@
+            bottomShadow = true,
+            onNavigationClick = onBackPressed,
+        )
+
+        LazyColumn(
+            modifier = modifier
+                .weight(1f, false)
+                .padding(vertical = 16.dp, horizontal = 16.dp),
+            verticalArrangement = Arrangement.spacedBy(16.dp),
+            contentPadding = PaddingValues(bottom = bottomOffset),
+        ) {
+            item {
+                PasswordField(
+                    password = passwordCurrent,
+                    isError = false,
+                    placeholder = stringResource(LR.string.profile_current_password),
+                    enabled = true,
+                    imeAction = ImeAction.Next,
+                    onImeAction = { passwordNewFocusRequester.requestFocus() },
+                    isCreatingAccount = false,
+                    focusRequester = passwordCurrentFocusRequester,
+                    onUpdatePassword = { viewModel.updatePwdCurrent(it) },
+                )
+            }
+            item {
+                PasswordField(
+                    password = passwordNew,
+                    isError = false,
+                    placeholder = stringResource(LR.string.profile_new_password),
+                    enabled = true,
+                    imeAction = ImeAction.Next,
+                    onImeAction = { passwordConfirmFocusRequester.requestFocus() },
+                    isCreatingAccount = false,
+                    focusRequester = passwordNewFocusRequester,
+                    onUpdatePassword = { viewModel.updatePwdNew(it) },
+                )
+            }
+            item {
+                Column {
+                    PasswordField(
+                        password = passwordConfirm,
+                        isError = false,
+                        placeholder = stringResource(LR.string.profile_confirm_new_password),
+                        enabled = true,
+                        imeAction = ImeAction.Done,
+                        onImeAction = { keyboardController?.hide() },
+                        isCreatingAccount = false,
+                        focusRequester = passwordConfirmFocusRequester,
+                        onUpdatePassword = { viewModel.updatePwdConfirm(it) },
+                    )
+                    TextH40(
+                        modifier = Modifier.padding(horizontal = 16.dp),
+                        text = errorMessage,
+                        color = MaterialTheme.theme.colors.support05,
+                    )
+                }
+            }
+            item {
+                RowLoadingButton(
+                    text = stringResource(LR.string.profile_confirm),
+                    onClick = onFormSubmit,
+                    isLoading = state is ChangePasswordState.Loading,
+                    enabled = !invalidPasswordCurrent && !invalidPasswordNew && !invalidPasswordConfirm,
+                )
```

Comment:
It's very minor, but the new version has changed the button padding, and the text boxes don't have the labels at the top after they have been entered. I don't think the labels matter if it's not supported in Compose.

| Before | After |
| --- | --- |
|  ![Screenshot_20240611_160131](https://github.com/Automattic/pocket-casts-android/assets/308331/daadc060-1696-41f2-9ab5-a9b058f65e60) | ![Screenshot_20240611_154812](https://github.com/Automattic/pocket-casts-android/assets/308331/5d5d02be-671e-48e5-ad69-2470326dfda2) |

Reply:
Thanks for your feedback. I've just fix the extra padding here [1d7f99c](https://github.com/Automattic/pocket-casts-android/pull/2312/commits/1d7f99caaf46b85da3ea1b9d20b07d81770ea4e8)

For the label I would say we are good since this matches with the already converted Change Email screen. I am going to merge this PR, but if you feel we need to update the label I can put another PR

Reply:
There is a big difference in my opinion between this screen and the email screen. On email screen there are 2 text fields that are easily distinguished by icons. This screen have 3 text fields all with the same icon, which makes it hard to read and infer the context of the fields.

Reply:
Got it! With the same icon makes harder to know the filed. I agree with you all